book.json

```diff
@@ -1,34 +0,0 @@
-{
```

Comment:
If we choose to continue building the documentation using gitbooks then this is probably required

Reply:
Put it back in.